### DISCUSSION TITLE
Scrolling is choppy on Mac

### DISCUSSION THREAD

**Discussion by marlonjames71** [notaapp/talk#282] [Feature Requests]
A simple one, but I've noticed recently that when scrolling in a note, the scrolling is noticeably chopping and it's definitely not as smooth as some native apps. My brother noticed it too, by the way.

I hope Nota is able to smooth out the scrolling as it's pretty noticeable.

Thanks!

**Comment by astoilkov** [comment]
I will look into this.

What machine are you using? Thanks.

Also, does it depend on the file or is it happening in all files?

**Reply by marlonjames71** [reply]
Thank you! I notice it on all files. I was using a 2018 MBP with i9 and the latest macOS. My brother has noticed the same thing even on the new M1 Max MBP.

**Comment by astoilkov** [comment]
I've improved the scrolling performance. The next release (in a few days) will include the improvement. Let's see how much of an improvement you will notice. You can write here after you test the new release.

**Reply by GraphicOwls** [reply]
I've noticed the same thing, so I'm really siked to see the improvements! 🙏

**Reply by marlonjames71** [reply]
Awesome! Thank you! Looking forward to the improvement as well!

**Comment by Arbiter313** [comment]
I’ve noticed this as well so thanks for the quick response!

**Comment by astoilkov** [comment]
As many people wrote here and were experiencing this, I continue to test the scrolling performance. I'm noticing something and I want to verify it with other people. Do you experience worse scrolling performance in files with images?

Thanks!

**Reply by GraphicOwls** [reply]
Whoa, yes I do actually. Now, that I just tried it, I do notice it. Much worse with images.

**Reply by marlonjames71** [reply]
Just tested and yes, the scrolling performance is significantly reduced when images are added to the file.

**Comment by astoilkov** [comment]
Now that 0.38.1 released, do you see a difference in the scrolling performance?

Note that I still haven't improved on the problem with images. However, I've improved the general scrolling performance which reduced the image problem a lot as well (at least for me).

**Reply by GraphicOwls** [reply]
Noticeably better, yes! That's awesome! 🤙 Thank you.

**Reply by Arbiter313** [reply]
Damn, have actually noticed a significant difference thank you!

**Reply by marlonjames71** [reply]
A lot better. Definitely noticeable. Thanks a ton!!

**Reply by astoilkov** [reply]
Awesome to hear that. Thank you all for your feedback.

I will be improving it even more but I'm happy that it got better.
